### PR TITLE
Revert Benchmark to SPY

### PR DESCRIPTION
**Pull request by wildart**
Reverted benchmark to SPY.

#### Description
Set benchmark symbol to SPY.

#### Related Issue
#2260

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Non-functional change (xml comments/documentation/etc)

#### Checklist:

- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** [document](https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md).
- [ ] I have added tests to cover my changes. 
- [ ] All new and existing tests passed.
- [x] My branch follows the naming convention `bug-<issue#>-<description` or `feature-<issue#>-<description>`

### CODE REVIEW

**Comment by mchandschuh** [comment]
@wildart -- how has this been tested?

**Comment by wildart** [comment]
I ran the subset of existing tests which executed on travis - they are fine. But, of course all regression tests failed.

**Comment by mchandschuh** [comment]
@wildart -- I had suspected that to be the case -- we can't merge this unless all the regression tests are passing.

You can set the configuration value `"regression-update-statistics": true` to automatically update the regression statistics. We'll need to also run these on a separate machine to ensure the values are correct.

File: Algorithm.CSharp/BasicTemplateFrameworkAlgorithm.cs [hunk 1]
```diff
@@ -82,25 +82,25 @@ public override void OnOrderEvent(OrderEvent orderEvent)
         /// </summary>
         public Dictionary<string, string> ExpectedStatistics => new Dictionary<string, string>
         {
-            {"Total Trades", "3"},
+            {"Total Trades", "19"},
```
Comment (mchandschuh) [inline Algorithm.CSharp/BasicTemplateFrameworkAlgorithm.cs]: Still doesn't feel right seeing the number of trades change due to changing the benchmark. I think this is an indication that something isn't completely right here.

**Comment by mchandschuh** [comment]
Closing for now while we work on the [Data Feed Architecture Streamling Project](https://github.com/QuantConnect/Lean/projects/5). This change highlighted some deficiencies in the architecture -- we'll come back to this following architecture improvements.

**Pull request closed without merging**